pkg/install/billing.go

```diff
@@ -0,0 +1,41 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+)
+
+func (i *Installer) createBillingRecord(ctx context.Context) error {
+	return cosmosdb.RetryOnPreconditionFailed(func() error {
```

Comment:
shouldn't be applicable/necessary?

Reply:
Don't think so, seems I overcomplicated the operation